examples/ippool/ippool.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: cluster.x-k8s.io/v1alpha4
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Please revert this file back for now since this is not the only place (we have to do same for infrastructure resource) where we have to update it. We will tackle updating the examples and docs to v1beta1 API once we are on track.

Reply:
@mboukhalfa  It is still there, can you remove your change please